### PR TITLE
fix: Decode decrypted response

### PR DESCRIPTION
**Pull request by Exelord**
fixes: https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues/455

### CODE REVIEW

File: pytapo/__init__.py [hunk 1]
```diff
@@ -563,7 +563,13 @@ def performRequest(self, requestData, loginRetryCount=0):
             encryptedResponse = responseData["result"]["response"]
             encryptedResponse = base64.b64decode(responseData["result"]["response"])
             try:
-                responseJSON = json.loads(self.decryptResponse(encryptedResponse))
+                decodedResponse = self.decryptResponse(encryptedResponse).decode("utf-8")
```
Comment (JurajNyiri) [inline pytapo/__init__.py:566]: I wonder why this was not needed on other cameras, I will run some tests and check code later to see if it doesn't cause any issues.

File: pytapo/__init__.py [hunk 2]
```diff
@@ -563,7 +563,13 @@ def performRequest(self, requestData, loginRetryCount=0):
...
+                
+                # Check if the response is wrapped with double curly braces
+                if decodedResponse.startswith('{{') and decodedResponse.endswith('}}'):
+                  decodedResponse = decodedResponse.replace('{{', '{').replace('}}', '}')
```
Comment (JurajNyiri) [inline pytapo/__init__.py:570]: Calling replace would replace `{{` and `}}` inside as well, which is not what we want. 
I suggest just trimming 1 character from the start and one from the end.

**Review by JurajNyiri — CHANGES_REQUESTED** [review]
Thank you! See my comments please, I will test this later and we can merge after.

File: pytapo/__init__.py (hunk 2 above)
Comment (Exelord) [reply pytapo/__init__.py:570]: good catch

Comment (Exelord) [reply pytapo/__init__.py:570]: I don't know python, so I would be glad if you could modify it further

Comment (JurajNyiri) [reply pytapo/__init__.py:570]: Put this in please on line 570 instead of what is there now:

`decodedResponse = decodedResponse[1:-1]`

Comment (JurajNyiri) [reply pytapo/__init__.py:570]: Hopefully this fix will not be needed though and we can get the hub to respond as expected, see https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues/455#issuecomment-1817484699 lets continue there

**Comment by JurajNyiri** [comment]
Closing

**Pull request closed without merging**